cmd/backup-manager/app/util/util.go

```diff
@@ -351,7 +351,7 @@ func GetBRArchiveSize(meta *kvbackup.BackupMeta) uint64 {
 
 // GetBRMetaData get backup metadata from cloud storage
 func GetBRMetaData(ctx context.Context, provider v1alpha1.StorageProvider) (*kvbackup.BackupMeta, error) {
-	s, err := NewStorageBackend(provider)
+	s, err := util.NewStorageBackend(provider, &util.StorageCredential{})
```

Comment:
just a small suggestion, here will use `util.StorageCredential{}` and will be set by `awsConfig.WithCredentials(cred.awsCred)`, and will set nil to aws api, maybe we can just set nil and don't set to aws api?

Reply:
My first version is set nil instead of &util.StorageCredential{}, it looks &util.StorageCredential{} more suitable.

Reply:
ok